### PR TITLE
Initial Import

### PR DESCRIPTION
**Pull request by quartzjer**
This fills in much of the draft template and imports some pre-existing introductory text provided by Srinath.

### CODE REVIEW

File: draft-setty-cfrg-spartan-incubation.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "TODO - Your title"
-abbrev: "TODO - Abbreviation"
+title: "Spartan High-speed zkSNARKs (incubation)"
```
Comment (srinathsetty) [inline draft-setty-cfrg-spartan-incubation.md:2]: Do we want the title to be a bit broad? For example SNARK-based verifiable credentials?

File: draft-setty-cfrg-spartan-incubation.md [hunk 2]
```diff
@@ -33,15 +33,37 @@ informative:
...
+
+An attractive aspect of our approach is that we can retrofit the aforementioned privacy properties to existing verifiable credentials issued with standard signature schemes (e.g., ECDSA over secp256k1 or P-256), with minimal resource overheads and engineering effort. Specifically, an issuer issues credentials as before with a standard signature scheme (e.g., ECDSA) so it is unmodified and incurs no overheads to support privacy features. Deriving a derived credential and verification of a derived credential happens on the credential holder and the verifier side, respectively. Specifically, the credential presentation protocol flow is as follows:
+* The issuer digitally signs a set of attribute-value pairs using its private key and sends ( to the holder, where is a digital signature (e.g., ECDSA signature).
```
Comment (srinathsetty) [inline draft-setty-cfrg-spartan-incubation.md:64]: A typo? "sender ("

File: draft-setty-cfrg-spartan-incubation.md [hunk 3]
```diff
@@ -33,15 +33,37 @@ informative:
...
+* The holder with produces a derived credential where is the disclosed subset of attribute-value pairs and is a proof of knowledge of such that: (i) is valid under the issuer’s public key for , and (ii) is a subset of .
```
Comment (srinathsetty) [inline draft-setty-cfrg-spartan-incubation.md:65]: Is there a typo here?

File: draft-setty-cfrg-spartan-incubation.md (hunk 1 above)
Comment (quartzjer) [reply draft-setty-cfrg-spartan-incubation.md:2]: This is just a starting point, we'll definitely be updating the title in the future including renaming the draft itself.

File: draft-setty-cfrg-spartan-incubation.md (hunk 2 above)
Comment (quartzjer) [reply draft-setty-cfrg-spartan-incubation.md:64]: This is your original text ;) IMO let's just merge it as-is and then make an editorial pass?

File: draft-setty-cfrg-spartan-incubation.md (hunk 3 above)
Comment (quartzjer) [reply draft-setty-cfrg-spartan-incubation.md:65]: (same as above)

File: draft-setty-cfrg-spartan-incubation.md (hunk 2 above)
Comment (srinathsetty) [reply draft-setty-cfrg-spartan-incubation.md:64]: The original text had symbols. Perhaps they were lost when turned into markdown?

**Pull request merged**